cargo-dylint/tests/ci.rs

```diff
@@ -534,6 +535,63 @@ fn update() {
     });
 }
 
+#[test]
+fn verify_registered_lints() {
+    let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let examples_dir = manifest_dir.join("../examples");
+
+    let folders = ["general", "supplementary"];
```

Comment:
```suggestion
    const CATEGORIES: &[&str] = &["general", "supplementary"];
```
Nit. I think "categories" is more descriptive than "folders". (Also, I think "folder" is technically a Windows term.)